deploy/roles/spark/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+spark_tar_file: "spark-1.5.2-bin-without-hadoop.tgz"
+spark_mirror: "http://apache.claz.org/spark/spark-1.5.2/spark-1.5.2-bin-without-hadoop.tgz"
+spark_src_dir: "/usr/local/src"
+spark_conf_dir: "/etc/spark"
+spark_usr_parent_dir: "/usr/lib"                                                            #spark extraction folder 
```

Comment:
@olivermel This one got missed too, can you align the comments to the left, above the code they refer to, so they won't get missed?